WaarpR66/src/main/java/org/waarp/openr66/configuration/FileBasedElements.java

```diff
@@ -146,6 +146,22 @@
    * SERVER HTTPS PORT
    */
   public static final String XML_SERVER_HTTPSPORT = "serverhttpsport";
+  /**
+   * SERVER PORT
+   */
+  public static final String XML_SERVER_IPS = "serverips";
+  /**
+   * SERVER SSL PORT
+   */
+  public static final String XML_SERVER_SSL_IPS = "serversslips";
+  /**
+   * SERVER HTTP PORT
+   */
+  public static final String XML_SERVER_HTTP_IPS = "serverhttpips";
+  /**
+   * SERVER HTTPS PORT
+   */
+  public static final String XML_SERVER_HTTPS_IPS = "serverhttpsips";
```

Comment:
Maybe we can name those tags `*addresses` instead of `*ips` by similarity with the existing `restaddress`

Reply:
Agree